gammapy/data/event_list.py

```diff
@@ -532,11 +532,11 @@ def plot_energy_hist(self, ax=None, ebounds=None, **kwargs):
         from ..spectrum import CountsSpectrum
 
         if ebounds is None:
-            emin = np.min(self['ENERGY'].quantity)
-            emax = np.max(self['ENERGY'].quantity)
+            emin = np.min(self.energy.value) * self.energy.unit
```

Comment:
Can you just change to this here?
```
emin = np.min(self.energy)
```
or
```
emin = self.energy.min()
```
?

I think the np.min function works fine with Quantity objects, no? At least this works for me:
```
import astropy.units as u
import numpy as np
np.min([1,2] * u.m)
```